index.dd

```diff
@@ -110,6 +108,31 @@ $(SCRIPT
     var rouletteIndex = Math.floor(Math.random() * examples.length);
     var rouletteChild = examples[rouletteIndex];
     rouletteChild.style.display = "block";
+
+    // build a list of the titles of all examples and add an option chooser to
+    // allow switching between them
+    var titles = Array.prototype.map.call(examples, function(e) {
+        return e.getElementsByClassName("your-code-here-title")[0].innerText;
+    });
+    var sel = document.createElement("select");
+    Array.prototype.forEach.call(titles, function(title, i) {
+        var el = document.createElement("option");
+        el.value = i;
+        el.innerText = title;
+        sel.appendChild(el);
+    });
+    sel.selectedIndex = rouletteIndex;
+    sel.addEventListener("change", function(e) {
+        examples[rouletteIndex].style.display = "none";
+        rouletteIndex = sel.selectedIndex;
+        examples[rouletteIndex].style.display = "block";
+    });
```

Comment:
The event handler could have been added later in the main JS file, but it's just 3 lines so no matter.